test/test_registration.py

```diff
@@ -84,18 +84,18 @@ def test_registration(self):
             cmd = RegisterCommand()
             cmd.main(['register', '--force', '--username', 'admin', '--password', 'admin', '--org', 'admin'])
 
-    @patch('subscription_manager.managercli.EntCertActionInvoker')
-    def test_activation_keys_updates_certs_and_repos(self, mock_entcertlib):
-        self.stub_cp_provider.basic_auth_cp = Mock('rhsm.connection.UEPConnection', new_callable=StubUEP)
-        self._inject_mock_invalid_consumer()
-
-        cmd = RegisterCommand()
-        mock_entcertlib = mock_entcertlib.return_value
-        self._inject_ipm()
-
-        cmd.main(['register', '--activationkey=test_key', '--org=test_org'])
-        self.mock_register.register.assert_called_once()
-        mock_entcertlib.update.assert_called_once()
+    #@patch('subscription_manager.managercli.EntCertActionInvoker')
```

Comment:
Why should we comment out this test? Is it not longer necessary? If it is no longer necessary let's remove it entirely. Otherwise we should leave it uncommented.

Reply:
I broke a few tests (with the same error) so I commented out all but one so I could figure out the fix.  I thought I got them all.